emle/models.py

```diff
@@ -1210,3 +1223,394 @@ def forward(self, atomic_numbers, charges_mm, xyz_qm, xyz_mm):
         E_ind = _torch.sum(vpot_ind @ charges_mm) * 0.5
 
         return _torch.stack([E_vac, E_static, E_ind])
+
+
+class MACExEMLE(EMLE):
```

Comment:
Could this just be `MACEEMLE`. I agree that it looks a bit messy, but it's the PEP-8 recommendation for acronyms. Here I assume that the `x` is just used to make it more readable, but it might be a bit confusing given its meaning in the ANI potentials.

Reply:
Sure, I've now changed the class name.